### DISCUSSION TITLE
Keycloak Integration with OKTA

### DISCUSSION THREAD

**Discussion by raviassesment** [keycloak/keycloak#23138] [Q&A]
Hi,

 We are trying to integrate keycloak with OKTA. We have configured the using OIDC following this documentation. https://ultimatesecurity.pro/post/okta-oidc/. When we are trying to logout from keycloak we are getting 400 from OKTA.

"Your request resulted in an error. The
'post logout redirect uri' parameter must be a Logout redirect URI in the client app settings:xxxxxxxxxxx"

**Comment by justin-tay** [comment]
You probably need to set the `Sign-out redirect URIs` as described at https://developer.okta.com/docs/guides/sign-users-out/android/main/.

![87929ad0cdef04d2975b18603ddff4108f42ea9e_2_1035x406](https://github.com/keycloak/keycloak/assets/49700559/0d44d4ee-72ba-4dd4-9361-cd4d9b64a382)
